gratipay/renderers/jinja2_html.py

```diff
@@ -0,0 +1,25 @@
+import aspen_jinja2_renderer as base
+
+from markupsafe import escape as htmlescape
+
+
+class HTMLRenderer(base.Renderer):
+    def render_content(self, context):
+        # Extend to inject HTML-escaping i18n functions.
+        _, ngettext = context['_'], context['ngettext']
+        context['_'] = lambda *a, **kw: _(*a, **dict(kw, escape=htmlescape))
+        context['ngettext'] = lambda *a, **kw: ngettext(*a, **dict(kw, escape=htmlescape))
+        return base.Renderer.render_content(self, context)
```

Comment:
Done in https://github.com/gratipay/gratipay.com/commit/1394e2da0afe8566be8e5593d5fab7c5c4295e45.